src/modifier.ts

```diff
@@ -167,3 +173,113 @@ function pairwiseMerge(modifiers1: Modifiers, modifiers2: Modifiers) {
+  else if (inputModifiers.length === 0) return;
+  else if (!baseModifier) {
+    return inputModifiers
+      .filter(
+        (mod1) =>
+          !inputModifiers.some((mod2) => mod2 !== mod1 && mod1.includes(mod2))
+      )
+      .forEach((baseMod) =>
+        printModtrace(
+          inputModifiers.filter((mod) => mod.includes(baseMod)),
+          baseMod
+        )
+      );
+  }
+
+  const htmlOutput = cliExecuteOutput(`modtrace ${baseModifier}`);
+  // The list of matched modifiers that mafia returns
+  const modtraceModifiers = Array.from(
+    htmlOutput.match(RegExp("(>)(.*?)(</td>)", "g")) ?? []
+  )
+    .map((s) => s.slice(1, -5))
+    .slice(2);
+
+  if (
+    !modtraceModifiers.some(
+      (modifier) => modifier.toLowerCase() === baseModifier.toLowerCase()
+    )
+  ) {
+    return print(
+      `Could not find exact string match of ${baseModifier} in ${inputModifiers.toString()}`,
+      "red"
+    );
+  }
+
+  const initialVal =
+    baseModifier.toLowerCase() === "familiar weight"
+      ? (() => {
+          const wt = familiarWeight(myFamiliar());
+          print(`[Familiar Weight] Base weight (${wt})`);
+          return wt;
+        })()
+      : 0;
+  const modifierVals = new Map(
+    modtraceModifiers.map((modifier) => [modifier, initialVal])
+  ); // Maps modifier name to its value
+  const lowerCaseModifiers = inputModifiers.map((modifier) =>
+    modifier.toLowerCase()
+  );
+
+  Array.from(htmlOutput.match(RegExp("<tr>(.*?)</tr>", "g")) ?? [])
+    .slice(1)
+    .map((s) => s.slice(4, -5))
+    .forEach((s) => {
+      const rowArr = Array.from(
+        s
+          .replace(RegExp("></td>", "g"), ">0</td>")
+          .match(RegExp("(>)(.*?)(</td>)", "g")) ?? []
+      ).map((s) => s.slice(1, -5));
+      const rowName = rowArr[1];
+      rowArr
+        .slice(2)
+        .filter((e, idx) => idx % 2 === 0)
+        .forEach((e, idx) => {
+          const val = parseInt(e);
+          modifierVals.set(
+            modtraceModifiers[idx],
+            (modifierVals.get(modtraceModifiers[idx]) ?? 0) + val
+          );
+          if (
+            val !== 0 &&
+            lowerCaseModifiers.includes(modtraceModifiers[idx].toLowerCase())
+          ) {
+            print(`[${modtraceModifiers[idx]}] ${rowName} (${val.toFixed(1)})`);
+          }
+        });
+    });
+
+  const total = sumNumbers(
+    modtraceModifiers.map((modifier) => {
+      if (lowerCaseModifiers.includes(modifier.toLowerCase())) {
+        let modVal = modifierVals.get(modifier) ?? 0;
+        if (
+          have($effect`Bow-Legged Swagger`) &&
+          modifier.includes("Weapon Damage")
+        ) {
+          print(`[${modifier}] Bow-Legged Swagger (${modVal.toFixed(1)})`);
+          modVal *= 2;
+        }
+        print(`${modifier} => ${modVal.toFixed(1)}`, "purple");
+        return modVal;
+      } else return 0;
+    })
+  );
+
+  print(`Total ${baseModifier}: ${total.toFixed(1)}`, "blue");
```

Comment:
We should let ourselves control the print colour, maybe; blue shows up terribly on darkmode